python/ray/util/client/server/server.py

```diff
@@ -30,12 +30,14 @@
 from ray.util.client.server.dataservicer import DataServicer
 from ray.util.client.server.logservicer import LogstreamServicer
 from ray.util.client.server.server_stubs import current_server
+from ray.ray_constants import env_integer
 from ray.util.placement_group import PlacementGroup
 from ray._private.client_mode_hook import disable_client_hook
 
 logger = logging.getLogger(__name__)
 
-TIMEOUT_FOR_SPECIFIC_SERVER_S = 30
+TIMEOUT_FOR_SPECIFIC_SERVER_S = env_integer("TIMEOUT_FOR_SPECIFIC_SERVER_S",
```

Comment:
Is this used for the idle server? So if there is a connection, it won't be used, right?

Reply:
Correct, this is only for a server that has **no clients connected**!